src/System.Private.CoreLib/shared/System/IO/Path.Windows.cs

```diff
@@ -210,13 +211,13 @@ public static bool IsPathRooted(ReadOnlySpan<char> path)
         // and "\\server\share" (a UNC path for a given server and share name).
         // The resulting string is null if path is null. If the path is empty or
         // only contains whitespace characters an ArgumentException gets thrown.
-        public static string GetPathRoot(string path)
+        public static string? GetPathRoot(string? path)
         {
             if (PathInternal.IsEffectivelyEmpty(path.AsSpan()))
                 return null;
 
             ReadOnlySpan<char> result = GetPathRoot(path.AsSpan());
```

Comment:
I wonder why you didn't have to use the `!` operator in this call but you had to do the the following, is something i this call affecting its nullability?

Reply:
Ah, I get it now, `AsSpan` is not an instance method, it is an extension method so it can be called with null...